### DISCUSSION TITLE
RouterOS (MikroTik) Script highlighting

### DISCUSSION THREAD

**Discussion by Dex811** [massCodeIO/massCode#170] [Ideas]
Hi,

first of all thanks for your work, I started using massCode a few days ago and it looks great!

My first thought (I have a few more in the queue ;-)) is is it possible to add highlighting support for RouterOS (MikroTik) Scripting language?

**Comment by antonreshetov** [comment]
In general, I have no plans yet to add more syntax highlights. Ace editor provides a [tool](https://github.com/ajaxorg/ace/wiki/Importing-.tmtheme-and-.tmlanguage-Files-into-Ace) to convert `.tmlanguage`. If there is a `.tmlanguage` for RouterOS, you can convert it to support it in the Ace editor and make a PR.

**Reply by Dex811** [reply]
OK, I found https://github.com/Kentzo/MikrotikScript/blob/master/MikrotikScript.tmLanguage so there is a way, but I'm much more of an admin than a coder, so I have to ask some of my friends for help that's OK

**Reply by antonreshetov** [reply]
@Dex811 [v3.0.0-beta.1](https://github.com/massCodeIO/massCode/releases/tag/v3.0.0-beta.1) supports Mikrotik. 

![image](https://user-images.githubusercontent.com/17480987/182824757-24aa0399-784d-41d4-8a9e-263c55f44fae.png)

**Reply by Dex811** [reply]
That's simply cool! Thank you sir! ;-)
